client/src/component/Sponsors/sponsors-logos.jsx

```diff
@@ -22,31 +26,55 @@ class Sponsors extends React.Component {
                 <img className="asucm" src={ASUCM} alt="ASUCM" /> {/*ASUCM logo and link*/}
                  </a> 
                  </div>
-            <div className = "column">
+            <div className = "top">
             <p><a href = "https://digitalocean.com/"> {/*goes to digital ocean site*/}
                  <img className = "digitalocean" src={DigitalOcean} alt="Digital Ocean" /> {/*Digital Ocean logo and link*/}
             </a></p>
             </div>
 
-            <div className = "column">
+            <div className = "top">
              <a href = "https://roche.com/" > {/*goes to roche site*/}
                  <img className = "roche" src= {roche} alt="Roche"/> {/*Roche logo and link*/}
                  </a>
                  </div>
             </div>
 
-            <div className = "bottom"> {/*bottom row for the display*/}
+            <div className = "2row"> {/*bottom row for the display*/}
                 <div className= "password">
             <a href= "https://1password.com/" > {/*goes to 1 password site*/}
                 <img className= "password" src = {password} alt="1Password"/> {/*1 paswword logo and link*/}
                 </a>
+                
+                <div className = "middle">
+                    <a href = "https://www.makeschool.com/">
+                        <img className = "makeschool" src = {makeschool} alt = "Make School" />
+                    </a>
+                </div>
+                <div className = "middle">
+                    <a href = "https://www.stickeryou.com/">
+                        <img className = "stickeryou" src = {stickeryou} alt = "Sticker You" />
+                    </a>
+                </div>
                 </div>
             
-                <div className= "bottom">
+                <div className= "3row">
             <a href = "https://www.sketch.com/"> {/*goes to sketch site*/}
                  <img className = "sketch" src = {sketch} alt="Sketch" /> {/*Sketch logo and link*/}
             </a>
-                         </div>
+               
+                 <div className = "bottom">
+                <a href = "https://balsamiq.com/">
+                        <img className = "balsamiq" src = {balsamiq} alt= "Balsamiq" />
+            </a>
```

Comment:
![image](https://user-images.githubusercontent.com/15200155/68064526-fdf05680-fcd9-11e9-9c7a-30e64037e0d9.png)
This looks scrunched up. do you think you can find a better image for this company?

Reply:
sorry, I will just adjust the width size